package.json

```diff
@@ -11,6 +11,7 @@
   "workspaces": [
     "@here/*"
   ],
+  "bin": "oauth-requester",
```

Comment:
Looks like it should be in  @here/olp-sdk-authentication/package.json ?

Reply:
Whoops... Sorry. Fixed :)